sv_pipeline/genome/load_data_tests.py

```diff
@@ -261,27 +354,17 @@ def test_subset_mt(self, mock_logger, mock_get_remap, mock_get_sample):
         ]
         mock_logger.info.assert_has_calls(calls)
 
-    @mock.patch('sv_pipeline.genome.load_data.load_gencode')
-    def test_annotation(self, mock_load_gencode):
```

Comment:
Remove the test for the `utility function`. It will be covered in the tests for the main test.